src/awfy.js

```diff
@@ -345,8 +345,6 @@ Object.entries(JSBENCH_TESTS).forEach(([testKey, test]) => {
   });
 });
 
-const VISUAL_METRICS = ['SpeedIndex', 'ContentfulSpeedIndex', 'PerceptualSpeedIndex'];
```

Comment:
Just an idea, but you could keep the array and only populate it with `SpeedIndex`, so that it would be easier to add other metrics?

Reply:
But maybe the simplified code is better.
All good :)